pkg/sidecar/sidecar.go

```diff
@@ -84,16 +89,48 @@ func Run(argv []string, version string) int {
 
 	klog.Infof("CSI driver name: %q", rt.DriverName)
 
-	// TBD May need to exposed metric HTTP end point
-	// here because the wait for the CSI driver is open ended.
+	rt.MetricsManager = metrics.NewMetricsManager()
+	wg := &sync.WaitGroup{}
```

Comment:
I'm not sure I understand the use of `wg` here. There is no call that I see to `wg.Add` nor any call to `wg.Wait`.